JAICore/jaicore-ml/src/main/java/ai/libs/jaicore/ml/core/dataset/serialization/ArffDatasetAdapter.java

```diff
@@ -222,6 +244,15 @@ private static boolean tryParseInt(final String string) {
 		}
 	}
 
+	/**
+	 * deserialising one instance of data (1 line)

```

Comment:
Parses a single instance of an arff file containing values for each attribute given.